src/models/connect.js

```diff
@@ -58,4 +60,24 @@ export default class Connect {
   newAccount() {
     // this.authorize() -> this.token()
   }
+
+  async gmailAccount({ name, emailAddress, scopes, googleClientId, googleClientSecret, googleRefreshToken }) {
```

Comment:
so I also tried to do an async function for `newAccount` and I kept getting this error when compiling: `Uncaught ReferenceError: regeneratorRuntime is not defined` . From this github issue, looks like it is solvable by making changes to the babel config file: https://github.com/babel/babel/issues/7824 . Did yours compile successfully as is though? I wonder if I did something wrong 🤔